py3status/modules/rate_counter.py

```diff
@@ -7,101 +7,157 @@
     config_file: file path to store the time already spent
         and restore it the next session
          (default '~/.i3/py3status/counter-config.save')
+    format: output format string
+        (default 'Time: {days} day {hours}:{mins:02d} Cost: {total}')
+    format_money: output format string
+        (default '{price}$')
     hour_price: your price per hour (default 30)
     tax: tax value (1.02 = 2%) (default 1.02)
 
+Format placeholders:
+    {days} The number of whole days in running timer
+    {hours} The remaining number of whole hours in running timer
+    {mins} The remaining number of whole minutes in running timer
+    {secs} The remaining number of seconds in running timer
+    {subtotal} The subtotal cost (time * rate)
+    {tax} The tax cost, based on the subtotal cost
+    {total} The total cost (subtotal + tax)
+    {total_hours} The total number of whole hours in running timer
+    {total_mins} The total number of whole minutes in running timer
+
+Money placeholders:
+    {price} numeric value of money
```

Comment:
Hmm, How we document this sort of stuff is a question.  I'm happy with this for now but maybe we need to decide as some other modules also do this 'subformatting'